docker_console/web/engines/base/tests.py

```diff
@@ -31,24 +31,25 @@ def docker_codecept(self, cmd='', parallel=False):
             selenium_container_removed = True
 
         if not selenium_container_id or selenium_container_removed:
-            run_cmd('docker run -d --name selenium-test-%s %s %s %s'
-                    % (self.docker.base_alias, self.docker._get_links(), self.docker._get_hosts(),
+            run_cmd('docker run -d --name selenium-test-%s %s %s %s %s'
+                    % (self.docker.base_alias, self.docker.get_env_file(),
```

Comment:
czemu to get_env_file zawsze dodajesz? to chyba powinno byc tylko wtedy kiedy jest ta opcja uzywana --with-env?

Reply:
Funkcja get_env_file() dodaje wpis tylko gdy jest podany parametr w wierszu poleceń.

https://github.com/droptica/docker-console/blob/master/docker_console/web/engines/base/docker.py#L234

Pomysł zaczerpnąłem z tego miejsca https://github.com/droptica/docker-console/blob/master/docker_console/web/engines/base/docker.py#L228